src/chapters/enums/intro.rst

```diff
@@ -0,0 +1,346 @@
+Introduction to Enums
+=====================
+
+.. index:: ! enumeration types
+
+Many statically-typed programming languages provide a feature called **enumeration types**, or **enum types** for short. Enum types are special classes that should have only one of a set of fixed values, such as days of the week. 
+
+Imagine that we wanted to add a field to ``Event`` to specify which day of the week an event takes place. You might start by creating a new field:
+
+.. sourcecode:: java
+
+   @NotBlank
+   private String dayOfWeek;
+
+As defined here, a user could submit any of the following values representing Wednesday:
+
+- ``"Wednesday"``
+- ``"wednesday"``
+- ``"Wed"``
+- ``"WED"``
+- ``"W"``
+
+Accepting data like this leads to many complications. For example, searching for all events taking place on Wednesday would need to account for all of these variations. And what happens if a user misspells "Wednesday"? 
+
+.. index:: select
+
+When filling out forms on the web, you are used to seeing ``select`` elements use to represent fields like this.
```

Comment:
use should be "used". alternatively you could say - 
"When filling out forms on the web, you have undoubtedly encountered ``select`` elements that represent fields like this:"